test/syscalls/linux/proc.cc

```diff
@@ -511,6 +513,87 @@ TEST(ProcSelfAuxv, EntryValues) {
   EXPECT_EQ(i, proc_auxv.size());
 }
 
+
+// Just open and read a part of /proc/self/mem, check that we can read an item.
+TEST(ProcSelfMem, Basic) {
+  auto memfd =
+      ASSERT_NO_ERRNO_AND_VALUE(Open("/proc/self/mem", O_RDONLY));
+  char input[] = "hello-world";
+  char output[sizeof(input)];
+  ASSERT_THAT(pread(memfd.get(), output, sizeof(output), (off_t) (input)),
+              SyscallSucceedsWithValue(sizeof(input)));
+  ASSERT_STREQ(input, output);
+}
+
+// Perform read on /proc/[pid]/mem from same task group.
+TEST(ProcSelfMem, TaskGroup) {
```

Comment:
Nit: This doesn't test /proc/self/mem. Maybe call the suite ProcPidMemTest instead?